webtau-core/src/main/java/org/testingisdocumenting/webtau/persona/Persona.java

```diff
@@ -16,24 +16,35 @@
 
 package org.testingisdocumenting.webtau.persona;
 
+import org.testingisdocumenting.webtau.utils.StringUtils;
+
 import java.util.Collections;
 import java.util.Map;
 import java.util.function.Supplier;
 
 public class Persona {
-    private static final Persona defaultPersona = new Persona("");
+    private static final Persona defaultPersona = new Persona("", Collections.emptyMap(), true);
     private static final ThreadLocal<Persona> currentPersona = ThreadLocal.withInitial(() -> defaultPersona);
 
     private final String id;
     private final Map<String, Object> payload;
+    private final boolean isDefault;
 
     public Persona(String id) {
         this(id, Collections.emptyMap());
     }
 
     public Persona(String id, Map<String, Object> payload) {
+        this(id, payload, false);
+    }
+
+    private Persona(String id, Map<String, Object> payload, boolean isDefault) {
```

Comment:
oh, this may conflict with the things in the other PR I have. I have `DEFAULT_PERSONA_ID` defined there...

Reply:
Yeah, I didn't want to add this isDefault property really.  However, I couldn't just have the id is null or empty check in the c'tor because that resulted in the default persona creation throwing 😁

Reply:
sorry I didn't notice it is a private constructor, should be good.